zerver/openapi/zulip.yaml

```diff
@@ -8906,6 +8906,23 @@ paths:
               - 2
               - 3
           example: 1
+        - name: show_unread_counts
+          in: query
+          description: |
+            This is used to configure in which streams does user wants to see unread messages count in left sidebar.
+
+            - 1 - All Streams
+            - 2 - Unmuted Streams
+            - 3 - No Streams
```

Comment:
Streams => streams in all of these; Zulip uses "Sentence case" for capitalization everywhere.